src/notation/view/widgets/editstaff.ui

```diff
@@ -1031,10 +1031,10 @@
         </size>
        </property>
        <property name="toolTip">
-        <string>Go to previous staff</string>
+        <string>Apply and go to previous staff</string>
        </property>
        <property name="accessibleName">
-        <string>Go to previous staff</string>
+        <string>Apply and go to previous staff</string>
```

Comment:
Question: is an accessibleName needed when there is a toolTip (with the same content? Wouldn't screen readers read it twice? @shoogle ?